### ISSUE TITLE
Getting Couldn't download release. Error: 599 Internal Exception

### ISSUE THREAD

**Issue by librasteve** [Raku/App-Rakubrew#78]
- What's the output of `rakubrew rakubrew-version`?:
  - rakubrew v37 Build type: macos OS: darwin
- Are you using rakubrew in `env` or in `shim` mode? (run `rakubrew mode` to find out):
  - env
- Where is rakubrew installed? (run `rakubrew home` to find out):
  - ~/.rakubrew

here's what I tried...

```
raku-Contact/bin > rakubrew download moar-2023.12
Downloading http://rakudo.org/dl/rakudo/rakudo-moar-2023.12-01-macos-arm64-clang.tar.gz
Couldn't download release. Error: 599 Internal Exception
```

EDIT: rakubrew build and build-zef worked fine (for my immediate needs ... so low priority for me)

**Comment by 2colours** [comment]
I'm getting the same error on Debian 12 bookworm.

**Comment by patrickbkr** [comment]
That's strange. There was a problem with the rakubrew.org website, but that should be unrelated. The URL given in the OP works for me at the moment. Did you try to download that file manually? Did that fail? Or was it only the rakubrew command that failed?

**Comment by librasteve** [comment]
seems like I had x86 rakubrew installed from a while back - so uninstall/install rakubrew fixed this - now works fine

**Comment by 2colours** [comment]
Uh, I got this error on amd64, following the usual instructions... I don't think this can be closed.

**Comment by librasteve** [comment]
ok -reopened

based on what I did to fix this issue, you may want to check that you have all the following:

native = amd64 | arm64 depending on your CPU

native perl (via which perl and then file xx the result)
native raku (via which raku and then file xx the result)
native rakubrew  (via rakubrew rakubrew-version)

I fixed by uninstalling rakubrew completely (per instructions on rakubrew.org) and then reinstalling (after updating my perl via curl -L -o download/perl-precomp.tar.gz https://github.com/skaji/relocatable-perl/releases/download/5.36.0.1/perl-darwin-arm64.tar.gz, untar, sh - since I am on M1)

**Comment by 2colours** [comment]
Well, I'm clueless what changed but now it worked for me from the same machine, with supposedly the very same setup as just a couple of days ago. And when I tried it a couple of days ago, I tried on the old VPS as well (which had an older Debian, not sure if 10 or 11) and it worked at the same time... now the old VPS is gone and suddenly it works on the new VPS.

I'd suspect there was indeed something with the server, no better idea.
